updated centos6 cmdlet

kraghu306 opened the pull request:
The existing bash script -> sudo sed -i 's/AutoUpdate.Enabled=n/AutoUpdate.Enabled=y/g' /etc/waagent.conf doesn't work as the AutoUpdateEnabled will be still uncommented as the format is "# AutoUpdate.Enabled=y".

azurecla commented:
Hi __@PedroPerezMSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

maafiri commented:
Not after November 6 :) 
MACs will persist across stop-deallocate and relaunch of VM's by then !
So I think waiting for another week without changing the doc shouldn't hurt anyone

PedroPerezMSFT commented:
Good news! Closing this PR then :)

kraghu306 committed:
updated centos6 cmdlet

The existing bash script -> sudo sed -i 's/AutoUpdate.Enabled=n/AutoUpdate.Enabled=y/g' /etc/waagent.conf doesn't work as the AutoUpdateEnabled will be still uncommented as the format is "# AutoUpdate.Enabled=y".
